src/main/java/org/jenkinsci/plugins/registry/notification/EnvContributor.java

```diff
@@ -46,6 +51,7 @@
 @Extension
 @Restricted(NoExternalUse.class)
 public class EnvContributor extends EnvironmentContributor {
+    private static final Logger logger = Logger.getLogger(EnvContributor.class.getName());
```

Comment:
where is this used?

Reply:
I bet I used it for debugging output at some point - will remove.